### PR TITLE
i18n support

### PR DESCRIPTION
**Pull request by jevanlingen**
Instead of using the name of the route as title, use the name as key for the i18n translation file.

You can override custom breadcrumbs, by using the i18nTitle:

```javascript
  breadCrumb: {
    i18nTitle: 'custom.i18n.key'
  }
```

### CODE REVIEW

**Comment by abhilashlr** [comment]
Do we need to take in i18n translations as part of breadCrumb? It is dependent on individual projects isn't  it? 

Passing i18n value from route to breadCrumb.title would solve this issue.

**Comment by jevanlingen** [comment]
Of course you could pass the i18n value from the route, as described in https://github.com/poteto/ember-crumbly/issues/69. But then you have to add a breadcrumb title to all of your routes. In my case, almost all route names are functional as breadcrumb titles.

By adding native i18n support to ember-crumbly, I just have to choose logical names for my routes + add the translation and it works out of the box.

--
In other words:

> For example, the route foo/bar/baz/1 will generate the following breadcrumb: Foo > Bar > Baz > Show. In most cases, this won't be exactly how you'd like it.

Well I do, but as my application needs multi language support, I just need to translate the route names.

**Comment by abhilashlr** [comment]
Wouldn't adding it for your project alone suffice?

Like create bread-crumbs.js and bread-crumbs.hbs in your components path?

Rather than doing it generically for breadcrumbs component.

File: addon/components/bread-crumb.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ export default Component.extend({
   layout,
   tagName: 'li',
   classNameBindings: ['crumbClass'],
+  i18n: false,
```
Comment (mike-north) [inline addon/components/bread-crumb.js]: Is there a conventional way of doing this yet? Maybe @jasonmit ([ember-intl](https://github.com/jasonmit/ember-intl) author) has some advice for things like this?

Comment (jasonmit) [inline addon/components/bread-crumb.js]: Based on the current implementation, I don't see why any change is needed to this lib.  `title` can be the translation key and you can implement the block form of the component.

```
{{#bread-crumbs outputStyle="bootstrap" linkable=true as |component crumb|}}
  {{#bread-crumb route=crumb breadCrumbs=component}}
      {{t crumb.title}}
  {{/bread-crumb}}
{{/bread-crumbs}}
```

I haven't used this library, it's just my initial reaction looking at the public API.

Comment (jasonmit) [inline addon/components/bread-crumb.js]: Or, a Mixin/base route that is responsible for setting `breadCrumb` in a lifecycle method.

```js
export default Ember.Mixin.create({
  i18n: Ember.inject.service(),

  init() {
    this._super(...arguments);

    if (!this.breadCrumb) {
      this.breadCrumb = {};
    }
  },

  afterModel() {
    const key = get(this, 'i18nTitle');

    if (key) {
      const breadCrumb = get(this, 'breadCrumb');

      Object.assign(breadCrumb, {
        title: get(this, 'i18n').t(key)
      });
    }

    return this._super(...arguments);
  }
});
```

**Comment by jevanlingen** [comment]
Yeah @jasonmit  could also work. Furthermore, I made a private fork of `ember-crumbly`, so this PR is no longer needed.

Thx all!
